shell/edit/provisioning.cattle.io.cluster/AgentConfiguration.vue

```diff
@@ -215,7 +186,7 @@ export default {
 </script>
 
 <template>
-  <div v-if="value && Object.keys(value).length">
```

Comment:
Why did you remove this condition? I am sure it was added with a specific purpose...